server/src/common/adresses.js

```diff
@@ -33,7 +33,8 @@ async function findCommune(codeInsee, codePostal, label) {
   }
 
   return buildAdresse({
-    label,
+    label: label,
+    //label_geojson: label,
```

Comment:
```suggestion
    label,
```

Il me semble que ça peut être suffisant ?